modules/renter/hostdb/consts.go

```diff
@@ -52,15 +52,15 @@ var (
 	// hostCheckupQuantity specifies the number of hosts that get scanned every
 	// time there is a regular scanning operation.
 	hostCheckupQuantity = build.Select(build.Var{
-		Standard: int(200),
+		Standard: int(10000),
 		Dev:      int(6),
 		Testing:  int(5),
 	}).(int)
 
 	// scanningThreads is the number of threads that will be probing hosts for
 	// their settings and checking for reliability.
 	maxScanningThreads = build.Select(build.Var{
-		Standard: int(40),
+		Standard: int(50),
```

Comment:
We need someone with a Mac to try this out when the maxScanningThreads value has been set to 200. I think we would run into the 'too many open files' error due to all of the open network connections, but I'm not sure. If it doesn't cause problems, going up to 200 threads would definitely help scanning go faster.

Reply:
I guess for the RC we can actually try putting it at a high value like 200 and see if anyone has issues.

Reply:
on the other hand, that's a move that will potentially break master, something we probably shouldn't be doing.